spec/lib/node_rpc_client_spec.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+RSpec.describe NodeRpcClient do
+  let!(:node) { create(:node) }
+
+  before do
+    stub_jrpc_request('options_prober.show.probers', node.rpc_endpoint, { logger: be_present }).and_return([{ id: 1 }])
```

Comment:
put return value to let, for example : `let(:jrpc_response)`

and use it in expectation